tsconfig.json

```diff
@@ -22,6 +22,6 @@
       "@/*": ["./*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "components/MusicPlayer/index.tsx", "components/MusicPlayer/Player.tsx"],
```

Comment:
It's really weird that you need to add these specific files to this list. You might not need them here at all, or you might need to move some types to a better place.